src/Raven.Server/Documents/Indexes/MapReduce/OutputToCollection/OutputReduceToCollectionCommand.cs

```diff
@@ -318,6 +329,12 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
             return _reduceDocuments.Count;
         }
 
+        private static unsafe bool DocumentsBinaryEqual(BlittableJsonReaderObject oldDocument, BlittableJsonReaderObject newDocument)
+        {
+            return oldDocument.Size == newDocument.Size &&
```

Comment:
Shouldn't we use `oldDocument.Equals(newDocument)` ?

Reply:
If the document size is different, I assume that the contents are different.
Since we build this document, I expect it to be with the same structure if it hasn't changed.